.travis.yml

```diff
@@ -1,16 +1,16 @@
-# vim: noai:ts=2:sw=2:ft=yaml:
+env:
+  global:
+  - secure: yh7EteqkcgDVgnMzuGb1rOtvMVP0iytjYG6qbaxDxv2zCgFllVWtMNiiBu9J79WXkLYA+QXDDb6oKeNAWeDo8o9rZQdiRgXtB0/SJ8ivKeFv3D1/52X8RqpJjpMWLT83rHWGYi2RDnEDU0MbxAR1XT4K9lRMN44U2VowNb2+h1+eCQG33plgECeqH9duGdcII75xf6v/dbta7AoDKuYGGzmOxU31CgK9wCoOT1/33K0UjrAhRxyu7NstFRWe6Exdq9RwG4Pk5BuJN5yWonb1OUeIGCVmO37D4Io/61tZUtTb3XIveVzYB8OA3H6ny+1QAsCaVqvOBzE8BKOxAtnx8lF70l7prKuJ1w+NiMB+cGDi2Kso+GhEvbhvmObuQ07Cyc9J6TOOqg2eLlx88HvTCQHrD+HKdytoD0c1TgXBsFtBoUk/H46mOhP97o6lgJ45n9EwcoOnBp1pwAqcnDVIenZxJaHi61nPZxMTUmrJKyO6MfFiJbV5c8wPmK6Ilg+M/44GuEr9Gj7GplZRv1z21YU6EyAeBfNemt5oFLzNF8Lb+KlpD5GKiRzZBfcHdd/eSw3DP2E+6WQLSvYviwrZxtYSLFjwPwfIVqmGN2Q3yCFRSdO2upSl43QGg6XaKyHFZDwuQ8y1HVC9VG6pz4qOpkSr6GuV17/d9x4k3fZ6RCk=
+  - secure: ojZXqjvCIqIbXJN13fma6rMH9A0y4b0B1Z9g/hG5ll3Dw76C6GnoxaH5tzAQuUEa3PgPG6oLsgmKr0drMZ4x6YESSVzLq2JIDUfct/qrYQ4CsUB9IjqO/8S2hxxMYsNbfLars34s6MTzNvReqMIMrxDPTEwc84a6jnmvIaFTTaqN4jQHrspZXcLD479muf1++YFnUgdobBKeS0XofZ/umRtW4thHv/yLSX2dcZ92Hz4Ad//YOromLunCfG1O6F7ggMEmEI4cuUp0J+TjBV4TdRnJz+T3ljAibgudSjUWVXSftxCei8CbwGuCK3f0UsQHS8qTBpylbt5iCFJEQf1MdFHi+yZY26jZqYZNboUmH7swPsA38je3Nd6siIXzPllf1B34O5xBubJ+CCqIBpNUe9cfJOqJ5EoibGYTuQmLdEwwY9OYd2dFvRtflSrKCHbz/8DgBmW+K8L1qGja2J069p0Ef7nuZpnhUqYB/mhExTb7hCKZeIKsVCIlmXzSTYqV8+HcyiQjol1eqehDjNrFaumVO5LX48APe1QpHClB/6uM74ZfCc4xa9UO0soBEYp8dOPsI3fbrRfVBI4gogv9/OLQFWTFZ1SKnjCYMcotFosoxp8x5ETki/VwGNCH0zobrFefnB4U4uX7NsrO4Vrx2zbur1/6NrhX1MChfGjZ1ok=
```

Comment:
Are you sure that `Lars34s6MT` shouldn't be `Lars34sGMT`?

Reply:
I just have whatever was generated/added by the `travis` binary (installed via `gem`).

So no... I'm not sure :)